content/md/cartridge-configuration/05-import-media-configuration.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 05-import-media-configuration
+themes: cartridge-configuration
+title: Configuration to import PIM media
+popular: false
+related: 01-where-configuration, 02-configure-PIM-API, 03-products-filter-configuration, 04-import-images-configuration, 06-mapping-configuration, 07-categories-configuration, 08-multi-storefront-configuration, 09-reference-entities
+---
+
+# First of all, what images are we talking about exactly?!
+
+Akeneo PIM since its 4.0 version (and then higher) offers to manage media with the Asset Manager.
+
+Since the 20.3.0 version, our SFCC connector allows the use of the Asset Manager for **Videos (Youtube & Vimeo)**, **PDF** and **Other** *(File or Media Link)*
+
+
+# How can I retrieve media from the "image" attributes?
+
+The media gathering process will be impacted by the way you have configured the cartridge in the [connector configuration page](01-where-configuration.html) regarding the way to manage images. If you need information on the images management please check the [image configuration page](04-import-images-configuration.html).
+
+# How can I retrieve media from the "Asset Collection" attributes?
+
+Please note that since the 20.3.0 version we have slightly changed the way to manage media (and then images): in order to not be constrained by the SharedLibraries limits anymore, whe have changed the way to store the media information: you will have the **asset collection and its content described as a custom product attribute**.
```

Comment:
```suggestion
Please note that since Akeneo Connector for SFCC version 20.3.0,  we have slightly changed the way to manage media (and then images): in order to not be constrained by the SharedLibraries limits anymore, we have changed the way to store the media information: you will have the PIM **asset collection and its content described as a SFCC custom product attribute**.
```